webui/src/Wizard/FinishStep.jsx

```diff
@@ -29,7 +29,8 @@ export function FinishStep({ oldConfig, newConfig }) {
 			</ol>
 			{((newConfig.elgato_plugin_enable && oldConfig.elgato_plugin_enable !== newConfig.elgato_plugin_enable) ||
 				(!newConfig.xkeys_enable && oldConfig.xkeys_enable !== newConfig.xkeys_enable) ||
-				(!newConfig.loupedeck_enable && oldConfig.loupedeck_enable !== newConfig.loupedeck_enable)) && (
+				(!newConfig.loupedeck_enable && oldConfig.loupedeck_enable !== newConfig.loupedeck_enable) ||
+				(!newConfig.shuttle_enable && oldConfig.shuttle_enable !== newConfig.shuttle_enable)) && (
```

Comment:
Actually, for consistent naming shouldn't this be like so?:
```suggestion
				(!newConfig.contour_enable && oldConfig.contour_enable !== newConfig.contour_enable)) && (
```